public/index.pug

```diff
@@ -433,7 +433,9 @@ html(lang=lang)
                   rel='noopener',
                   target='_blank'
                 ) #{ icon.license.type }
-              h2.grid-item__title #{ icon.localizedTitle || icon.title }
+              h2.grid-item__title.copy-button.copy-slug(
+                data-slug=`${ icon.slug }`
```

Comment:
We are currently extracting slug from `href` value of the icon SVG download button to avoid `data-` attributes using the function `iconHrefToSlug`. Would you mind to get the slug using that strategy instead and remove the `data-slug` attributes?

You can see an example of this at [this line (import)](https://github.com/simple-icons/simple-icons-website/blob/6f87c65f89c34f05e4937ec9e5aa4f6f251cf080/public/scripts/download-type.js#L2) and [these lines (usage)](https://github.com/simple-icons/simple-icons-website/blob/6f87c65f89c34f05e4937ec9e5aa4f6f251cf080/public/scripts/download-type.js#L58-L59). Just go to the parent traversing the DOM and locate the `a[download]` link node.

Reply:
Good point. Thank you!